### PR TITLE
User experience updates

### PR DESCRIPTION
**Pull request by AlecKap**
__x__ Wrote Tests __x__ Implemented __x__ Reviewed

## Necessary checkmarks:

- [x] All Tests are Passing in all environments

- [x] The code will run locally

## Type of change

- [ ] New feature
- [ ] Bug Fix
- [x] Refactor

## Description of Change:

    description: 
    - Change the redirection in the populi_transfer_controller create action so that instead of redirecting to the attendance_show page after transferring student attendances to populi, it redirects bak to the module_show page. 
    - Update populi_transfer_spec test to reflect new redirection to module_show page instead of attendance_show page.

## Requested feedback:

    I'd like feedback on...

## Check the correct boxes

- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [ ] No Tests have been changed
- [x] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [x] My code has no unused/commented out code
- [x] My code has no binding.pry's
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link: https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExemtjdzVpempjdTlkNHU1dzIzOTl2b2V4eGhkczk4NGd1MTM1M29wYiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ddZ2mYhxhh69wEpSVz/giphy.gif

### CODE REVIEW

**Review by domingusj — APPROVED** [review]

File: spec/features/attendances/populi_transfer_spec.rb [hunk 1]
```diff
@@ -57,6 +57,15 @@
     it 'has link to transfer student attendances to populi' do
       expect(page).to have_button("Transfer Student Attendances to Populi")
     end
+
+    it 'has a link to return to the attendance_show page' do
+      save_and_open_page
```
Comment (AlejandroLopez1992) [inline spec/features/attendances/populi_transfer_spec.rb]: Gotta remove this save and open page

**Review by AlejandroLopez1992 — CHANGES_REQUESTED** [review]
Nice user experience changes, I think the application will feel smoother with these path changes.

File: app/views/user/populi_transfer/new.html.erb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<br>
+<%= link_to sanitize("&larr; Back to Attendance"), attendance_path(@attendance) %>
```
Comment (AlejandroLopez1992) [inline app/views/user/populi_transfer/new.html.erb:2]: Could you explain what the sanitize link entails and why it's better to use it in this circumstance?

Comment (AlecKap) [reply app/views/user/populi_transfer/new.html.erb:2]: It helps protect the Rails application from Cross Site Scripting (XSS) attacks, validates the URL format, and prevents open redirects. This context probably doesn't warrant such security measures, but it's just a security practice I have been utilizing.

File: spec/features/attendances/populi_transfer_spec.rb (hunk 1 above)
Comment (AlecKap) [reply spec/features/attendances/populi_transfer_spec.rb]: Ok, I will remove this!

**Review by AlejandroLopez1992 — APPROVED** [review]

**Pull request merged**